zulipterminal/scripts/render_symbols.py

```diff
@@ -0,0 +1,60 @@
+import urwid
+
+from zulipterminal.config import symbols
+
+
+def exit_on_q(key: str) -> None:
+    if key in ('q', 'Q'):
+        raise urwid.ExitMainLoop()
+
+
+palette = [
+    ('header', 'light blue, bold', ''),
+    ('sub_header', 'white, bold', 'light blue'),
+]
+
+variables = vars(symbols)
+symbol_dict = {symbol: variables[symbol] for symbol in variables
+               if not symbol.startswith('__') and not symbol.endswith('__')}
+
+symbol_names_list = [urwid.Text(symbol, align='center')
+                     for symbol in symbol_dict]
+symbols_list = [urwid.Text(symbol_dict[symbol], align='center')
+                for symbol in symbol_dict]
+
+symbol_names_widget = urwid.LineBox(urwid.Pile(symbol_names_list),
+                                    title="Symbol Names",
+                                    title_attr='sub_header',
+                                    tlcorner='│', tline='─', lline='│',
+                                    trcorner='┬', blcorner='└', rline='│',
+                                    bline='─', brcorner='┴')
+symbols_widget = urwid.LineBox(urwid.Pile(symbols_list), title="Symbols",
+                               title_attr='sub_header',
+                               tlcorner='─', tline='─', lline='',
+                               trcorner='│', blcorner='─', rline='│',
+                               bline='─', brcorner='┘')
+
+symbols_display_box = urwid.Columns([
+    symbol_names_widget,
+    symbols_widget
+])
+
+line_box = urwid.LineBox(symbols_display_box,
+                         title="Render Symbols",
+                         title_attr='header',
+                         tlcorner='┌', tline='─', lline='',
+                         trcorner='┐', blcorner='', rline='', bline='',
+                         brcorner='')
+info_box = urwid.Text(('quit_text', u"Press 'q' to exit."), align='center')
+
+display_box = urwid.Pile([
+    line_box,
+    info_box
+])
+body = urwid.Filler(display_box, valign='middle')
+
+
+def main() -> None:
+    loop = urwid.MainLoop(urwid.Padding(body, left=30, right=30),
```

Comment:
I suggest we could try decreasing the padding, to avoid splitting of large symbol names in multiple lines.

Reply:
I've not checked the docs, but can't we leave the padding not specified and still get it centered?

Reply:
It would still be centered, but visually not very clean as it renders a large box and most of the space is left unused. I was initially at a padding of 40 on the right and left on Windows Terminal, and then reverted back to 30 after running it on the GNOME terminal on Ubuntu.